### ISSUE TITLE
Keep getting javascript error on mobile - Slider

### ISSUE THREAD

**Issue by CmdrMahesh** [fomantic/Fomantic-UI#1147]
# I'm getting javascript error while I'm using the fomantic UI with WordPress.

## Javascript error on slider's node movement in mobile.

hi, I tried to create a slider using fomantic-ui and this works fine on desktop but throws an error on mobile view in chrome.

as far as I know, this works well even in mobile but as I fused this to WordPress site it keeps throwing an error in the mobile view.

here is live site link : [Demo](http://101.53.136.134/demo/jewellery/product-category/diamond/?EO_WBC=1&BEGIN=diamond&STEP=1)

in-order to get an error toggle to mobile view and click on filter then carat and you can see a slider but when you try to change the slider's nodes position it will throw the following error in console.

```
Uncaught TypeError: Cannot read property '0' of undefined
    at Object.eventPos (semantic.min.js?ver=5.1.3:16114)
    at Object.down (semantic.min.js?ver=5.1.3:15703)
    at HTMLDivElement.<anonymous> (semantic.min.js?ver=5.1.3:15653)
    at HTMLDivElement.dispatch (jquery.js?ver=1.12.4:3)
    at HTMLDivElement.r.handle (jquery.js?ver=1.12.4:3)
```

**Comment by lubber-de** [comment]
I cannot reproduce using you demo site. Please try to use the unminified version `semantic.js` and reproduce the error (better: provide a link and **detailed steps** to reproduce)
With the error console output of the unminified js file, we probably can find out what might cause your issue.
As said, the current link works fine in chrome mobile view
![image](https://user-images.githubusercontent.com/18379884/68195579-6465c500-ffb7-11e9-8ce1-513465ac62ad.png)

**Comment by lubber-de** [comment]
MMh, ok, i can reproduce the error now (i really had to start sliding the thumb on mobile view.
However, as everything is packed into one js file i cannot figure out easily where it breaks.
Please provide a jsfiddle or at least use the unminified FUI Version, otherwise i dont know where to start for investigation

**Comment by CmdrMahesh** [comment]
@lubber-de pleas ignore the name of the file, the file is un-minified version of FUI and in console error is: 
```
Uncaught TypeError: Cannot read property '0' of undefined
    at Object.eventPos (semantic.min.js?ver=5.1.3:16114)
    at Object.down (semantic.min.js?ver=5.1.3:15703)
    at HTMLDivElement.<anonymous> (semantic.min.js?ver=5.1.3:15653)
    at HTMLDivElement.dispatch (jquery.js?ver=1.12.4:3)
    at HTMLDivElement.r.handle (jquery.js?ver=1.12.4:3)
```
so, assume from line number 16114 of semantic.js. Once again please ignore the file name.

**Comment by lubber-de** [comment]
oups... ok, thanks, we'll investigate

**Comment by CmdrMahesh** [comment]
I think this may be related to screen size as it works fine in desktop view and gets stuck in the mobile view, where the user triggers sliding actions.

**Comment by lubber-de** [comment]
@CmdrMahesh Finally found the reason and fixed it by #1149 
You are obviously using jquery 2. If you would switch to jquery 3 your issue is solved as well, but i fixed it in general in the above mentioned PR

**Comment by CmdrMahesh** [comment]
Thanks  @lubber-de, hope this gets added to main branch soon, so other get this fix too.
